pkgs/servers/asterisk/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "asterisk-${version}";
-  version = "13.6.0";
+  version = "13.9.1";
 
   src = fetchurl {
     url = "http://downloads.asterisk.org/pub/telephony/asterisk/asterisk-${version}.tar.gz";
```

Comment:
Can you change to `http://downloads.asterisk.org/pub/telephony/asterisk/releases/${name}.tar.gz` (Note the `releases/`, so the link remains working when they release a new version)?